week-4/D-methods/README.md

```diff
@@ -1,19 +1,70 @@
-### Goals
-- make objects even more useful and more closer to "real-life" by introducing object methods
-- explain how to get access to the current object reference (the `athlete` inside the the `drink` function in the example below)
+### Object methods
 
-The should be able to understand the following:
+Besides having specific properties, objects in the real world can also do things. For example, a computer can display something on the screen, a person can say their names etc... To translate this into the Javascript world, let's introduce object's methods. 
+A method is a function attached to a particular object.

```

Comment:
"Let's introduce object's methods" sounds a bit awkward. Perhaps these two sentences (from "To translate ...") could be shortened to something like "In Javascript we do this using 'methods'. A method is <blah blah>. Then maybe say that we have already been using predefined methods on strings (like `toUppercase()`) and on arrays.